data/newsletter/issue-260.markdown

```diff
@@ -4,7 +4,8 @@ This is a weekly summary of what's going on in its community.
 
 ## Featured
 
-undefined
+- [Of function instances and abstract syntax](https://www.danielbrice.net/blog/of-function-instances-and-abstract-syntax/)
```

Comment:
Would you like attribution? 
```suggestion
- [Of function instances and abstract syntax](https://www.danielbrice.net/blog/of-function-instances-and-abstract-syntax/) by Daniel Brice
```

Reply:
Yes, please. It just didn't occur to me earlier.